### DISCUSSION TITLE
Using Git with versioned Medley files

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1451] [Q&A]
When using Git to commit Medley files, such as Lisp sources or TEdit documents, how do you deal with the different versions of the files? Do you commit only whatever files the filesystem links Medley maintains point to? The [`.gitignore`](https://github.com/Interlisp/medley/blob/master/.gitignore) of Medley doesn't seem to ignore versions.

Any suggestions or recommended tools for dealing with versioned Medley files with Git?

**Comment by masinter** [comment]
Looking in the Medley repo .gitignore there is, line 31

# older versions

*~

which basically ignores all .~nn~ fiiles too.

**Reply by nbriggs** [reply]
and ignores editor backup files (from, for example, emacs) which is also a good thing.
